app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">FakeNews</string>
+	<string name="author">Author:</string>
+	<string name="title">Title</string>
```

Comment:
Ah nice! Looks like you created the label values already, you could reference these directly in in the article_view.xml layout for the `authorLabel` and `titleLabel` elements.